examples/pytorch/dimenet/config/dimenet.yaml

```diff
@@ -0,0 +1,33 @@
+name: "dimenet"
+
+model:
+  emb_size: 128
+  num_blocks: 6
+  num_bilinear: 8
+  num_spherical: 7
+  num_radial: 6
+  envelope_exponent: 5
+  cutoff: 5.0
+  num_before_skip: 1
+  num_after_skip: 2
+  num_dense_output: 3
+  # ['mu', 'alpha', 'homo', 'lumo', 'gap', 'r2', 'zpve', 'U0', 'U', 'H', 'G', 'Cv']
+  targets: ['U0']
+  # 'zeros' for mu, homo, lumo, and zpve; 'GlorotOrthogonal' for alpha, R2, U0, U, H, G, and Cv
+  output_init: 'GlorotOrthogonal'
```

Comment:
Do we expect users to employ different initialization methods? If not, I think you can use default initialization methods depending on the targets specified.